src/module.zig

```diff
@@ -59,6 +65,10 @@ pub fn Objects(comptime T: type) type {
 
             /// Global pointer to object relations graph
             graph: *Graph,
+
+            /// This is mach's way of determining which fields are updated and
+            /// need to trigger some other logic
+            updated: std.ArrayListUnmanaged(std.bit_set.DynamicBitSetUnmanaged) = .{},
```

Comment:
```
/// A bitset per-field used to track field changes. Only used if options.track_fields == true.
```

Reply:
We could make this field nullable and have it be null if track_fields == false? Not sure if that's better memory wise but maybe worth considering.

Reply:
Done